prt_partner_tweaks_access/tests/test_partner_tweaks_access.py

```diff
@@ -0,0 +1,129 @@
+from odoo.tests import tagged
+
+from ..consts import PREDEFINED_RULES
+from .common import PartnerTweaksAccessCommon
+
+
+@tagged("post_install", "-at_install", "standart")
+class TestPartnerTweaksAccess(PartnerTweaksAccessCommon):
+    def test_user_without_tweak_access(self):
+        """
+        Check that results differ for a user without tweaks and a user with tweaks.
+        """
+
+        result_1 = self.partner_model.with_user(self.user_without_tweaks_access).search(
+            []
+        )
+        result_2 = self.partner_model.with_user(self.user_with_tweaks_access).search([])
+
+        # Ensure that the search results are not equal for the two users
+        self.assertNotEqual(
+            result_1, result_2, msg="The search results should not be equal"
```

Comment:
They differ but how exactly?

Reply:
I fixed it
The users are different because the access settings are different